src/VscProcess.cpp

```diff
@@ -209,13 +235,29 @@ bool VscProcess::KeyString(KeyString::Request& req, KeyString::Response& res)
   return true;
 }
 
+bool VscProcess::vscSettingsSrv(GetVscSettings::Request  &req, GetVscSettings::Response &res)
+{
+  res.srv_ready = srv_ready;
+  res.serial_number = serial;
+  res.firmware_version = firmware;
+  res.radio_power_level = radio_power_level;
+
+  return true;
+}
+
 void VscProcess::processOneLoop(const ros::TimerEvent&)
 {
   // Send heartbeat message to vehicle in every state
   vsc_send_heartbeat(vscInterface, myEStopState);
 
   // Check for new data from vehicle in every state
   readFromVehicle();
+
+  if (have_serial && have_radio_power_level && have_firmware && !srv_ready)
```

Comment:
Looks like these are member variables. I'd recommend appending `_` to the end of them for consistency.

Reply:
The rest of the repo, including every other file doesn't follow this convention, so it would be out of the scope of this PR to change every other preexisting variable along with these as well. I believe we should opt to keep them as is to conform to the submodule's own formatting rules.